bundle/src/main/java/com/adobe/acs/commons/wcm/impl/PropertyMergePostProcessor.java

```diff
@@ -21,183 +21,257 @@
 import java.util.Collection;
 import java.util.Date;
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.LinkedHashSet;
 import java.util.List;
-import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.jackrabbit.JcrConstants;
 
 /**
  * ACS AEM Commons - Property Merge Sling POST Processor
  */
 @Component
 @Service
 public class PropertyMergePostProcessor implements SlingPostProcessor {
+
     private static final Logger log = LoggerFactory.getLogger(PropertyMergePostProcessor.class);
 
     private static final String AT_SUFFIX = "@PropertyMerge";
     private static final String ALLOW_DUPLICATES_SUFFIX = AT_SUFFIX + ".AllowDuplicates";
     private static final String TYPE_HINT_SUFFIX = AT_SUFFIX + ".TypeHint";
     private static final String IGNORE_PREFIX = ":";
+    private static final String ALL_TAGS = "merge-all-tags";
```

Comment:
Something about this naming doesnt pop out to me as it being a special operation.. which is it (vs a property name; even thought property name's SHOULD be camelCase). If we wind up having more special operations like this, it might be nice to have a clean convention .. maybe something like:

`Operation.MergeAllTags` 

.. WDYT?

Reply:
I think that anything long becomes hard to enter when working with the asset schema editor UI.  Though it is more coder-friendly, it becomes rather business user hostile and harder to remember.  I use hypens because I ultimately do a case-insensitive match on it to make it more user friendly.  There is a slim-to-nil chance of someone using this as a property name.

Reply:
If youre dead set on it, might as well remove "merge" and just have "all-tags", it's even less for the schema editor user to remember; "merge" is a bit of an implementation idiom. (couldve just as easily been "combine" or another synonym).

Reply:
Well all-tags is likely to be a name for a property.  What if we used a non-jcr friendly prefix like you did for :?  Like :@all-tags or something?

Reply:
surely a property `all-tags` would be unusual since thats what `cq:tags` would (should) be for? In a way the raison d'etre for this feature is to aggregate tags into cq:tags, since cq:tags is the only property that the rest of AEM knows about, but managing multiple namespaces of tags in a single property is bad UX.

`:@all-tags` is good with me; there is a clear indication that this is "special" about this vs someone being lazy when coming up a property name. 

Ultimately someone is gonna have to look this up one way or another..  seems simple, and good to me.